src/plots/polar/layout_defaults.js

```diff
@@ -55,6 +55,7 @@ function handleDefaults(contIn, contOut, coerce, opts) {
         // propagate the template.
         var axOut = contOut[axName] = {};
         axOut._id = axOut._name = axName;
+        axOut._traceIndices = subplotData.map(function(t) { return t.index; });
```

Comment:
In cartesian you use `t._expandedIndex` - should it be the same here?

Reply:
Good eye. Thanks!

Reply:
fixed and :lock: in https://github.com/plotly/plotly.js/pull/2860/commits/d233f3bbadc2aacd6539fd397c701cf7dd9e5796